config/initializers/pg_search.rb

```diff
@@ -0,0 +1,7 @@
+# See:
+# https://github.com/Casecommons/pg_search#trigram
```

Comment:
this is <https://github.com/Casecommons/pg_search#trigram-trigram-search>

Reply:
It is indeed, somehow that got cut off. Thanks. I fixed it locally now, will push when I update the settings.